crates/router/src/connector/coinbase/transformers.rs

```diff
@@ -32,10 +65,10 @@ pub struct CoinbasePaymentsRequest {
     pub cancel_url: String,
 }
 
-impl TryFrom<&types::PaymentsAuthorizeRouterData> for CoinbasePaymentsRequest {
+impl TryFrom<CoinbseRouterData<&types::PaymentsAuthorizeRouterData> for CoinbasePaymentsRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
-        get_crypto_specific_payment_data(item)
+    fn try_from(item:&CoinbseRouterData<&types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        get_crypto_specific_payment_data(item.RouterData)
```

Comment:
Hey @DhairyaMajmudar 

You have to change the definition of `get_crypto_specific_payment_data` such that it starts accepting `CoinbseRouterData<&types::PaymentsAuthorizeRouterData>` as an argument rather than passing `item.router_data`.

Please refer to this for more context similar change was made. https://github.com/juspay/hyperswitch/pull/2645/files

Reply:
I am a bit confused in this change can you pls. perform this task to remove the confusion

Reply:
Hey @DhairyaMajmudar 
I cannot perform this task for you, I have referred you with an already merged PR you just have to follow it and replicate it.

Basically you just have to change the type of the argument which the function is accepting.